frontend/src/components/QuestionCard/QuestionCard.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from "react";
+// nodejs library that concatenates classes
+import classNames from "classnames";
+// nodejs library to set properties for components
+import PropTypes from "prop-types";
+// @material-ui/core components
+import Card from "components/Card/Card.js";
+import CardBody from "components/Card/CardBody.js";
+import CardHeader from "components/Card/CardHeader.js";
+import Radio from "@material-ui/core/Radio";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import FiberManualRecord from "@material-ui/icons/FiberManualRecord";
+import Grid from '@material-ui/core/Grid';
+
+import { makeStyles } from '@material-ui/core/styles';
+import radoStyles from "assets/jss/material-kit-react/customCheckboxRadioSwitch.js";
+
+// core components
+const styles = {
+  textCenter: {
+    textAlign: "center"
+  },
+  textAlignLeft: {
+    textAlign: "left"
+  },
+  image: {
+    maxWidth: "100%",
+    height: "auto"
+  },
+  ...radoStyles
+};
+
+const useStyles = makeStyles(styles);
+
+export default function QuestionCard(props) {
+  const classes = useStyles();
+  const { className, question_id, description, picture_link, answers, index } = props;
```

Comment:
Console in Inspect Element says className and question_id are defined but never used.

Reply:
Deleted className but keep question_id because it might be useful when creating testing page